ai_agents/agents/ten_packages/extension/speechmatics_asr_python/extension.py

```
@@ -229,17 +233,23 @@ async def _handle_asr_result(
         start_ms: int = 0,
         duration_ms: int = 0,
         language: str = "",
+        metadata: dict = None,
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)